x-pack/plugins/code/server/lsp/request_expander.ts

```diff
@@ -75,6 +79,8 @@ export class RequestExpander implements ILanguageServerHandler {
         reject,
         startTime: Date.now(),
       });
+      if (this.log)
+        this.log.debug(`queued  a ${request.method} job for workspace ${request.workspacePath}`);
```

Comment:
nit: additional space in the log sentence, also add `{}` for the if block? not sure if the lint check will complain this.